CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 # airsonic/airsonic
 # -->
 
+## v10.4.2 - 22 Oct 2019
+
+### Fixes
+
+- Fix a migration issue on MariaDB (#1333)
```

Comment:
Since the changelog is written in markdown, maybe it's worth providing a link to the issue?

Reply:
Hum, the link is added by GitHub on the releases page anyway (and is useless without GitHub). I'll do that next time? :wink: